src/deluge/io/midi/midi_device.h

```diff
@@ -155,11 +163,10 @@ class MIDIDeviceUSBHosted final : public MIDIDeviceUSB {
 
 class MIDIDeviceUSBUpstream final : public MIDIDeviceUSB {
 public:
-	MIDIDeviceUSBUpstream(uint8_t portNum = 0) { portNumber = portNum; }
+	MIDIDeviceUSBUpstream(uint8_t portNum = 0) : MIDIDeviceUSB(portNum) {}
```

Comment:
Is the portnum still necessary after giving it a default value in the mididevice USB class?